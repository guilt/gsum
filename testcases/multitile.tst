Name: multitile
Description: Test 5M file with four-tile pattern, full file hash
File: multitile.txt 5M tile 1M 0x00,0x01,0x02,0x03
AlgoMatrix: default
