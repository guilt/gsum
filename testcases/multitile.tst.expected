Hash-default: 397e58c42741bda596daded7f27265bf99ff332b8915390063d4952f0a6305a9
